src/applications/claims-status/reducers/serialize.js

```diff
@@ -56,19 +82,20 @@ export const serializeClaim = claim => {
     associatedDocuments,
   );
 
+  const transformedUnassociatedDocuments = transformUnassociatedDocs(
+    filterAssociatedDocs(supportingDocuments),
+  );
+
+  const transformedAssociatedTrackedItems = transformAssociatedTrackedItems(
+    associatedTrackedItems,
+  );
+
   return {
     ...claim,
     attributes: {
       ...claim.attributes,
-      supportingDocuments: filterAssociatedDocs(supportingDocuments).map(
-        doc => {
-          return {
-            ...doc,
-            date: doc.uploadDate,
-          };
-        },
-      ),
-      trackedItems: associatedTrackedItems,
+      supportingDocuments: transformedUnassociatedDocuments,
+      trackedItems: transformedAssociatedTrackedItems,
```

Comment:
These variables are a bit wordy; if we don't find the labeling helpful, we can just run the relevant functions like so:
```js
      supportingDocuments: transformUnassociatedDocs(
        filterAssociatedDocs(supportingDocuments),
      ),
      trackedItems: transformAssociatedTrackedItems(
        associatedTrackedItems,
      ),
```

The result is the same regardless.